### PR TITLE
ICMSLST-598 Add Further Questions to OPT editing (phase one).

### PR DESCRIPTION
**Pull request by oskusalerma**
This adds the questions without file uploads.

### CODE REVIEW

File: web/domains/case/_import/opt/models.py [hunk 1]
```diff
@@ -47,6 +52,112 @@ class OutwardProcessingTradeApplication(ImportApplication):
...
 
-    # TODO: ICMLST-598 add Further Questions
+    fq_similar_to_own_factory = models.CharField(
+        max_length=3,
+        choices=YesNoNAChoices.choices,
+        null=True,
+        verbose_name="""
+            Do you manufacture goods which are similar to and at the same stage of
+            processing in your own factory within the EU as the products to be
+            re-imported? (Article 2 (2) (a) of Regulation (EC) No. 3036/94)""",
```
Comment (MattHolmes123) [inline web/domains/case/_import/opt/models.py:62]: Use one pair of double quotes and `()` to split on to multiple lines. e.g.
```
"description": (
    "You no longer need to type the part of the Firearms Act that applies to the"
    " item listed in this box. You must select it from the 'Licence for' section."
),
```

File: web/domains/case/_import/opt/models.py [hunk 2]
```diff
@@ -47,6 +52,112 @@ class OutwardProcessingTradeApplication(ImportApplication):
...
+        help_text=_ONCE_PER_YEAR,
+    )
+
+    fq_manufacturing_within_eu = models.CharField(
+        max_length=3,
+        choices=YesNoNAChoices.choices,
+        null=True,
+        verbose_name="""
```
Comment (MattHolmes123) [inline web/domains/case/_import/opt/models.py:70]: same for all of them

File: web/domains/case/_import/opt/views.py [hunk 3]
```diff
@@ -63,6 +61,44 @@ def edit_opt(request: HttpRequest, *, application_pk: int) -> HttpResponse:
...
+def edit_further_questions(request: HttpRequest, *, application_pk: int) -> HttpResponse:
+    with transaction.atomic():
+        application: OutwardProcessingTradeApplication = get_object_or_404(
+            OutwardProcessingTradeApplication.objects.select_for_update(), pk=application_pk
+        )
+
+        task = application.get_task(ImportApplication.IN_PROGRESS, "prepare")
+
+        # TODO: use check_application_permission
+        if not request.user.has_perm("web.is_contact_of_importer", application.importer):
```
Comment (MattHolmes123) [inline web/domains/case/_import/opt/views.py:75]: Did you mean to use check_application_permission as part of this commit or not?

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]

File: web/templates/web/domains/case/import/opt/edit-further-questions.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ {% import "forms/fields.html" as fields %}
+
+ {% block sidebar %}
+     {% include "partial/case/import/opt/sidebar.html" %}
+ {% endblock %}
+
+ {% block main_content %}
+   {% block task_details %}
+     {{ super() }}
+   {% endblock %}
```
Comment (MattHolmes123) [inline web/templates/web/domains/case/import/opt/edit-further-questions.html:12]: Do we need this block if we are just calling super()?

**Review by kevinetienne — APPROVED** [review]

**Pull request closed without merging**